### PR TITLE
Changelog and changes for version 1.2.0

### PR DESCRIPTION
**Pull request by davidperezgar**
Updated Changelog.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -68,6 +68,29 @@ In any case, passing the checks in this tool likely helps to achieve a smooth pl
 
 == Changelog ==
 
+= 1.2.0 =
+
+- The Plugin Check is now part of the automatic checks for plugin submissions on WordPress.org.
```
Comment (felixarntz) [inline readme.txt]:
```suggestion
* The Plugin Check is now part of the automatic checks for plugin submissions on WordPress.org.
```

**Review by felixarntz — APPROVED** [review]

File: readme.txt (hunk 1 above)
Comment (swissspidy) [reply readme.txt]: This line is not reallly a change in the plugin itself though, so maybe it could also be removed entirely?

Comment (mukeshpanchal27) [reply readme.txt]: Agree with @swissspidy

Comment (davidperezgar) [reply readme.txt]: ok

**Review by ernilambar — APPROVED** [review]

**Pull request merged**